src/hermione/commands/selectStory.ts

```diff
@@ -53,6 +53,10 @@ export function createSelectStory(storybookUrl: string, sessionStore: SessionSto
             }
         }
 
+        await this.waitUntil(() => this.execute(function() {
+            return Boolean(window.__HERMIONE_SELECT_STORY__);
+        }), {timeout: 3000, interval: 100, timeoutMsg: "Hermione addon is not connected to storybook config"});
+
```

Comment:
`previewAnnotations` sometimes need a little bit of time to init
This makes tests more stable